lib/cdo/contact_rollups.rb

```diff
@@ -301,6 +301,10 @@ def self.sync_contact_rollups_to_main(log_collector)
 
     log("#{Time.now} Completed. #{num_total} source rows processed. #{num_inserts} insert(s), #{num_updates} update(s), #{num_unchanged} unchanged.")
     log_collector.info("#{num_total} source rows processed. #{num_inserts} insert(s), #{num_updates} update(s), #{num_unchanged} unchanged.")
+  rescue StandardError => error
```

Comment:
You can use the `log_collector` here to capture the error right when it happens. By doing that we won't have to manually run the script later hopefully it will fail at the same step.

In addition, this will swallow the iteration error and the rest of the script will keep going without knowing there is a failure in the previous step. Should we re-raise here and stop processing?

Reply:
Good point.  I've fixed that.